cmd/action/internal/uri/uri.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021-present The Atlas Authors. All rights reserved.
+// This source code is licensed under the Apache 2.0 license found
+// in the LICENSE file in the root directory of this source tree.
+
+package uri
```

Comment:
don't create another package for one function. leave private in the `action` package.